### ISSUE TITLE
Bug in previous versions of physicsSystem

### ISSUE THREAD

**Issue by SAnsell** [SAnsell/CombLayer#30]
There has been a bug in some of the previous versions of CombLayer from about early April. I introduced a tidy up of the phyiscsSystem cut card, and made it such that a multi-particle different cut card was not possible. This was missed in the tests etc. It then got fixed without actually seeing that it was a problems while improving the overall physics card system -- however I noticed it on checking model version differences. 

  In short -- use the latest master and kbatMerge branches which are fixed.
